spec/unit/recipes/default_spec.rb

```diff
@@ -72,33 +72,26 @@
   context 'When a package install is specified' do
     let(:chef_run) do
       ChefSpec::ServerRunner.new do |node|
-        node.override['audit']['inspec_package'] = true
+        node.override['audit']['inspec_package']['source'] = 'http://path/to/fake.rpm'
```

Comment:
I think the inspec package installation tests should be moved into a `inspec_spec.rb` file under `spec/unit/recipes` since the best practice is for the unit test file name to be `<recipe_name>_spec.rb`.